core/src/Cabana_DeepCopy.hpp

```diff
@@ -126,6 +126,119 @@ inline void deep_copy(
     }
 }
 
+//---------------------------------------------------------------------------//
+/*!
+  \brief Create a mirror of the given AoSoA in the given memory
+  space. Same space specialization returns the input AoSoA.
+ */
+template<class Space, class SrcAoSoA>
+SrcAoSoA
+create_mirror_aosoa(
```

Comment:
inline all of these.

Reply:
done

Reply:
note that I used regular `inline` instead of `KOKKOS_INLINE_FUNCTION`